crates/sui-framework/sources/event.move

```diff
@@ -7,14 +7,4 @@ module sui::event {
     // TODO(https://github.com/MystenLabs/sui/issues/19):
     // restrict to internal types once we can express this in the ability system
     public native fun emit<T: copy + drop>(event: T);
-
```

Comment:
Why the change here?

Reply:
Same problem as with `test_scenario` - calling `emit` with a type parameter here was making the bytecode verifier angry...